sickbeard/server/web/core/base.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'base_url': urljoin(rh.request.protocol + '://' + rh.request.host,
+                                sickbeard.WEB_ROOT) +
+                          ('', '/')
+                          [urljoin(rh.request.protocol + '://' + rh.request.host, sickbeard.WEB_ROOT)[-1] not in '/'],
```

Comment:
@p0psicles @medariox this is already in develop with my changes and is working fine. If you want to change it I'd suggest opening a new PR.